swap_meet/item.py

```diff
@@ -1,2 +1,38 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, category=""):
+        self.condition = condition
+        self.category = category
+
+        # if no given id, generate uuid
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
```

Comment:
👍 Nicely done! We should use `is` for a `None` check like you do on line 6. 

One way we could condense this code is to write our condition to replace what `id` is pointing at if necessary, then assign the updated value to our attribute. 

Structuring code this way allows us to manage edge cases (when there is no assigned `id` yet) at the top and leave the focus of the logic unindented. 

```suggestion
        if id is None:
            self.id = uuid.uuid4().int

        self.id = id
```

Or you could use a ternary operator too:
```py
self.id = uuid4().int if id is None else id